java/openmldb-jdbc/src/test/java/com/_4paradigm/openmldb/jdbc/SQLRouterSmokeTest.java

```diff
@@ -367,9 +367,7 @@ public void testInsertPreparedState(SqlExecutor router) {
             insert = "insert into tsql1010 values(?, ?, ?, ?, ?);";
             PreparedStatement impl3 = router.getInsertPreparedStmt(dbname, insert);
             impl3.setLong(1, 1003);
-            impl3.setString(3, "zhejiangxx");
```

Comment:
Got, I think use `List` or `Map` more better instead of `int`, because we want to implement multiple set and save the last value. So this issue and PR may unnecessary

Reply:
Done, reverted